test/fairtools/_TestFairTools.cxx

```diff
@@ -71,323 +62,321 @@ template <class T> class _TestFairLoggerBase : public T
+        levelNames.erase(position + 1, levelNames.end());
+        return levelNames;
     }
 
-    void CheckScreenOutput(FairCaptureOutput handler, const std::vector<std::string>& v) {
-      int noLines = handler.GetNumberOfLines();
-      int expectedNumberOfLines = v.size();
-      EXPECT_EQ(expectedNumberOfLines,noLines);
+    void CheckScreenOutput(FairCaptureOutput handler, const std::vector<std::string>& v)
+    {
+        int noLines = handler.GetNumberOfLines();
+        int expectedNumberOfLines = v.size();
+        EXPECT_EQ(expectedNumberOfLines, noLines);
 
-      int minLines = std::min(expectedNumberOfLines,noLines);
+        int minLines = std::min(expectedNumberOfLines, noLines);
 
-      for (int i = 0; i < minLines; ++i) {
-        EXPECT_EQ( v[i], handler.GetCaptureLine(i) );
-      }
+        for (int i = 0; i < minLines; ++i) {
+            EXPECT_EQ(v[i], handler.GetCaptureLine(i));
+        }
     }
 
-    void CheckFileOutput(FairTestOutputHandler handler, const std::vector<std::string>& v) {
-      int noLines = handler.GetNumberOfLines();
-      int expectedNumberOfLines = v.size();
-      EXPECT_EQ(expectedNumberOfLines,noLines);
+    void CheckFileOutput(FairTestOutputHandler handler, const std::vector<std::string>& v)
+    {
+        int noLines = handler.GetNumberOfLines();
+        int expectedNumberOfLines = v.size();
+        EXPECT_EQ(expectedNumberOfLines, noLines);
 
-      int minLines = std::min(expectedNumberOfLines,noLines);
+        int minLines = std::min(expectedNumberOfLines, noLines);
 
-      for (int i = 0; i < minLines; ++i) {
-        EXPECT_EQ( v[i], handler.GetCaptureLine(i) );
-      }
+        for (int i = 0; i < minLines; ++i) {
+            EXPECT_EQ(v[i], handler.GetCaptureLine(i));
+        }
     }
 
-    bool CheckVerboseOutput(std::string logLevel, std::string outputString,
-                            std::string verboseLevel, std::string resultString
-                           ) {
-      // Example Output for logging level INFO and different verbosity levels
-      // verbosity Level:Output
-      // LOW    :[INFO   ] I am here.
-      // MEDIUM :[INFO   ] [_TestFairTools.cxx::LogNoArguments:78] I am here.
-      // HIGH   :[INFO   ] [05.12.2011 12:00:10] [_TestFairTools.cxx::LogNoArguments:78] I am here.
-      std::string regexString = "\\[" + logLevel + "[ ]*][ ]";
-      if (verboseLevel.compare("HIGH") == 0) {
-        regexString = regexString + "\\[\\d{2}\\.\\d{2}\\.\\d{4}[ ]\\d{2}:\\d{2}:\\d{2}][ ]";
-        regexString = regexString + "\\[.*::.*:\\d{2}][ ]";
-      }
-      if (verboseLevel.compare("MEDIUM") == 0) {
-        regexString = regexString + "\\[.*::.*:\\d{2}][ ]";
-      }
-      regexString = regexString + outputString + ".*";
-
-      const std::regex e(regexString);
-      return std::regex_match(resultString, e);
+    bool CheckVerboseOutput(std::string logLevel,
+                            std::string outputString,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'outputString' should be passed by const reference.](https://app.codacy.com/manual/FairRootGroup/FairRoot/pullRequest?prid=5323596)